cheetah/accelerator.py

```diff
@@ -62,6 +62,24 @@ def __init__(self, name=None, device="auto"):
         self.device = device
 
     def transfer_map(self, energy):
+        """
+        Necessary for the calculations in '__call__'.
```

Comment:
Maybe be be more specific along the lines of "Generates the element's transfer map that describes how the beam and its particles are transformed when travelling through the element.